jsettlers.main.swing/src/jsettlers/main/swing/SwingManagedJSettlers.java

```diff
@@ -79,6 +81,44 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 		generateContent(argsMap, content);
 	}
 
+	private static String getSelectedFolderName() {
+		
+		//- Info: Don't use JFileChooser(), some Windows User have problems caused by a NullPointerException in FileDialog()
+		//-   see issue #240: https://github.com/jsettlers/settlers-remake/issues/240
+		
+		
+		//- see: http://stackoverflow.com/questions/1224714/how-can-i-make-a-java-filedialog-accept-directories-as-its-filetype-in-os-x
+		JFrame frame = new JFrame();
+		
+		System.setProperty("apple.awt.fileDialogForDirectories", "true");
```

Comment:
apple.awt? I would really recommend not to use platform specific stuff. 

On Windows I'm unable to select a folder. When I select a file, I have to know that I have to select a file in the root folder of the Settlers 3 installation to make it work. However, I don't get this information from the dialog.